hpcflow/sdk/helper/helper.py

```diff
@@ -146,6 +187,44 @@ def restart_helper(
     start_helper(app, timeout, timeout_check_interval, watch_interval, logger=logger)
 
 
+def write_helper_args(
+    app,
+    pid,
+    timeout=DEFAULT_TIMEOUT,
+    timeout_check_interval=DEFAULT_TIMEOUT_CHECK,
+    watch_interval=DEFAULT_WATCH_INTERVAL,
+):
+    PID_file = get_PID_file_path(app)
+    try:
+        with PID_file.open("wt") as fp:
+            fp.write(f"pid = {pid}\n")
+        with PID_file.open("at") as fp:
```

Comment:
Could you remove this line, as discussed?

Reply:
Resolved in [e0569e4](https://github.com/hpcflow/hpcflow-new/pull/340/commits/e0569e4048b9228aea4504b4bb61e6e73569b5ef)